Ports/mrsh/patches/0003-Workaround-warnings.patch

```diff
@@ -0,0 +1,26 @@
+From bb874f39033650acc4cc4926afd066e8919f86ec Mon Sep 17 00:00:00 2001
+From: EWouters <6179932+EWouters@users.noreply.github.com>
+Date: Tue, 12 Apr 2022 18:01:44 +0200
+Subject: [PATCH 3/4] Workaround warnings
+
+---
+ main.c | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/main.c b/main.c
+index 0f433d2..577afde 100644
+--- a/main.c
++++ b/main.c
+@@ -14,7 +14,9 @@
+ #include <unistd.h>
+ #include "frontend.h"
+ 
++#ifndef __serenity__
+ extern char **environ;
++#endif
```

Comment:
Is this actually needed? Why would it complain about an `extern` definition of `environ`?

Reply:
This is the error is remedies:
```
main.c:17:15: error: redundant redeclaration of 'environ' [-Werror=redundant-decls]
   17 | extern char **environ;
      |               ^~~~~~~
In file included from main.c:14:
/Users/erik/git/serenity-master/Build/x86_64/Root/usr/include/unistd.h:31:15: note: previous declaration of 'environ' with type 'char **'
   31 | extern char** environ;
      |               ^~~~~~~
cc1: all warnings being treated as errors
```
I'll be more expressive in the commit message.